portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro

```diff
@@ -1972,7 +1972,9 @@ definition {
 	}
 
 	macro viewSwitchChecked {
-		FormFields.viewCheckboxChecked(fieldName = "${fieldName}");
+		AssertChecked.assertCheckedNotVisible(
```

Comment:
@leticiachiu, shouldn't we confirm if the switch is visible instead since the macro is named `viewSwitchChecked`?

Reply:
@natocesarrego as you can see, before, this macro was the same as `viewCheckboxChecked` and that macro uses `AssertChecked.assertCheckedNotVisible` too, the only difference is that I changed the path to `FormFields#SWITCH`

Reply:
```
macro viewCheckboxChecked {
		AssertChecked.assertCheckedNotVisible(
			key_fieldName = "${fieldName}",
			locator1 = "FormFields#CHECKBOX");
}
```

Reply:
@leticiachiu, considering the macro's legibility, let's use the following:
```
AssertChecked(
	key_fieldName = "${fieldName}",
	locator1 = "FormFields#CHECKBOX");
```
By default, if no function is invoked in the macro's call (`AssertChecked`) then the default function is invoked, e.g. `assertChecked`, as you can see [here](https://github.com/liferay/liferay-portal/blob/fbc6889b98c1c40185f6b136e0554be917c310b5/portal-web/test/functional/com/liferay/portalweb/functions/AssertChecked.function#L1).

Can you please adjust it and trigger `ci:forward`?

Thanks in advance. :slightly_smiling_face: